src/hooks/wallets/mpc/useMPCWallet.ts

```diff
@@ -33,12 +32,21 @@ export const useMPCWallet = (): MPCWalletHook => {
   const mpcCoreKit = useMPC()
   const onboard = useOnboard()
 
+  const isMFAEnabled = () => {
+    if (!mpcCoreKit) {
+      return false
+    }
+    const { shareDescriptions } = mpcCoreKit.getKeyDetails()
+
+    return !Object.entries(shareDescriptions).some(([key, value]) => value[0]?.includes('hashedShare'))
```

Comment:
`key` is not used

```suggestion
    return !Object.values(shareDescriptions).some((value) => value[0]?.includes('hashedShare'))
```

Reply:
This also duplicated the logic found in `useMFASettings`. It might be worth abstracting this to a helper function.

Reply:
It was not used here. I removed it from this hook.